<!-- -->
<launch>
  <arg name='host' default='localhost'/>
  <arg name='port' default='2000'/>
  <arg name="role_name" default="ego_vehicle" />
  <arg name="spawn_point" default="190,186.4,3,0,0,180"/>

  <param name="/carla/host" value="$(arg host)" />
  <param name="/carla/port" value="$(arg port)" />

  <node pkg="region_of_interest" type="lane_line_publisher.py" name="lane_line_publisher" output="screen">
    <param name="role_name" value="$(arg role_name)" />
    <param name="spawn_point" value="$(arg spawn_point)"/>
  </node>
</launch>
